tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "baseUrl": ".",
     "target": "es5",
     "lib": ["esnext"],
+    "strict": true,
```

Comment:
can't believe typescript doesn't enable this by default! It caught so much stuff